lib/msal-browser/src/cache/BrowserCacheManager.ts

```diff
@@ -708,7 +693,7 @@ export class BrowserCacheManager extends CacheManager {
             this.cacheConfig.cacheLocation === BrowserCacheLocation.LocalStorage
```

Comment:
This can be removed so we emit the LOGOUT always here (also the deprecation notice above)